main/ble_server.c

```diff
@@ -38,24 +39,24 @@
 #define SPP_PROFILE_NUM             1
 #define SPP_PROFILE_APP_IDX         0
 #define ESP_SPP_APP_ID              0x56
-#define DEVICE_NAME_GAP          "BLE_TO_ISOTP"    //The Device Name Characteristics in GAP
-#define SPP_SVC_INST_ID	            0
+#define DEVICE_NAME_GAP             "BLE_TO_ISOTP"    //The Device Name Characteristics in GAP
+#define SPP_SVC_INST_ID             0
 
 /// SPP Service
-static const uint16_t spp_service_uuid = 0xABF0;
+static const uint16_t spp_service_uuid = 0xABF0; // '0000abf0-0000-1000-8000-00805f9b34fb'
 /// Characteristic UUID
-#define ESP_GATT_UUID_SPP_DATA_RECEIVE      0xABF1
-#define ESP_GATT_UUID_SPP_DATA_NOTIFY       0xABF2
-#define ESP_GATT_UUID_SPP_COMMAND_RECEIVE   0xABF3
-#define ESP_GATT_UUID_SPP_COMMAND_NOTIFY    0xABF4
+#define ESP_GATT_UUID_SPP_DATA_RECEIVE      0xABF1 // '0000abf1-0000-1000-8000-00805f9b34fb'
+#define ESP_GATT_UUID_SPP_DATA_NOTIFY       0xABF2 // '0000abf2-0000-1000-8000-00805f9b34fb'
+#define ESP_GATT_UUID_SPP_COMMAND_RECEIVE   0xABF3 // '0000abf3-0000-1000-8000-00805f9b34fb'
```

Comment:
should we drop the command stuff entirely? i can for sure see the usecase for it but right now we have nothing. maybe a command to reset the whole device?

Reply:
I'd like to keep it for now, I was thinking about maybe writing a j2534 dll eventually and for that it will be nice to be able to add a few more commands (battery voltage read, version metadata info). could also be used to setup filters if we ever end up wanting those.